ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -788,8 +765,6 @@ public void onCompletion(ReadableStreamChannel routerResult, Exception routerExc
     public void onCompletion(String routerResult, Exception routerException) {
       callbackTracker.markOperationEnd();
       if (routerResult == null && routerException == null) {
-        frontendMetrics.routerCallbackError.inc();
-        callbackTracker.markCallbackProcessingEnd();
```

Comment:
if not here, where do we mark the processing as end (or in other words, update the processing time metric?